### PR TITLE
Load comments from Datastore

### PR DESCRIPTION
**Pull request by aroquev00**
The webapp now stores and loads all comments from Datastore, providing persistence to the data. The doGet method of the servlet was modified to query the comments from DS and send them to the client.

Link to deployment: https://aroquev-step-2020.uc.r.appspot.com/

### CODE REVIEW

File: aroquev/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,12 +30,25 @@
...
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    final ArrayList<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String comment = (String) entity.getProperty("commentBody");
+      // Timestamp is not used at the moment, but will help in next feature
```
Comment (brettapeters) [inline aroquev/src/main/java/com/google/sps/servlets/DataServlet.java:46]: `id` is also not used. You should remove the unused variables until they're needed.

Comment (aroquev00) [reply aroquev/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: aroquev/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -27,12 +30,22 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (seanjohnite) [inline aroquev/src/main/java/com/google/sps/servlets/DataServlet.java:31]: update TODO?

File: aroquev/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -27,12 +30,22 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    final ArrayList<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("commentBody");
```
Comment (seanjohnite) [inline aroquev/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Do these comments have any metadata? Are they just strings?

File: aroquev/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (aroquev00) [reply aroquev/src/main/java/com/google/sps/servlets/DataServlet.java:31]: Done.

File: aroquev/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (aroquev00) [reply aroquev/src/main/java/com/google/sps/servlets/DataServlet.java:44]: As for now they are, but will add info such as timestamp and other things and for that I plan to create a Comment class.

**Pull request merged**